src/components/new-request/search-components/business-lookup-fetch.vue

```diff
@@ -131,7 +131,8 @@ export default class BusinessLookupFetch extends Mixins(CommonMixin, SearchMixin
         this.setEntityTypeCd(this.corpTypeToEntityType(corpType))
       } else if (this.isSelectedXproAndRestorable) { // Check if XPRO and restorable
         this.setLocation(Location.CA)
-        this.setEntityTypeCd(this.getSearchBusiness.legalType)
+        const corpType = this.getSearchBusiness.legalType as unknown as CorpTypeCd
+        this.setEntityTypeCd(this.corpTypeToEntityType(corpType) || this.getSearchBusiness.legalType)
```

Comment:
Should you have conditional chaining and fallback to null here, same as below?

Reply:
Yes it should be, I will update. Looking over it again this appears to be the only place it is needed. It actually isn't needed below (or in jurisdiction.vue) since they are wrapped in with `if (this.getSearchBusiness)`.